template-for-recording/record_screen_and_upload.bat

```diff
@@ -173,16 +173,22 @@ echo.
 echo JAVA_VERSION=%JAVA_VERSION_INT_VALUE%
 
 set JAVA_VERSION_9=9
+set JAVA_VERSION_11=11
 
 if %JAVA_VERSION_INT_VALUE% lss %JAVA_VERSION_9% (
    echo "--- Pre-Java 9 detected ---"
    echo "Using DEFAULT_JVM_OPTS variable with value '%DEFAULT_JVM_OPTS%'"
 ) else (
    echo "--- Java 9 or higher detected (version %JAVA_VERSION_INT_VALUE%) ---"
-   set DEFAULT_JVM_OPTS=--illegal-access=warn --add-modules=java.xml.bind,java.activation %DEFAULT_JVM_OPTS%
-   echo "Adding JVM args to the DEFAULT_JVM_OPTS variable, new value set to '%DEFAULT_JVM_OPTS%'"
-   echo "--------------------------------------------------------------------------------------------------------------"
+   if %JAVA_VERSION_INT_VALUE% lss %JAVA_VERSION_11% (
+       set DEFAULT_JVM_OPTS=--illegal-access=warn --add-modules=java.xml.bind,java.activation %DEFAULT_JVM_OPTS%
+       echo "Adding JVM args to the DEFAULT_JVM_OPTS variable, new value set to '%DEFAULT_JVM_OPTS%'"
+   ) else (
+      echo "Using DEFAULT_JVM_OPTS variable with value '%DEFAULT_JVM_OPTS%'"
+      echo "Not using kill-switch or modules flags starting from this version of Java"
```

Comment:
How come this works for Java 11 but not for Java 9?

Reply:
What is the explanation for it?

Reply:
The answer to the question also relates to the comment below to unify the solution, see https://github.com/julianghionoiu/record-and-upload/issues/7#issuecomment-439677957

In both versions, if we include the dependency at build time, we won't need to use the flags.

Reply:
Just applied via 939732246ad6904ae03270a0c661fb302cba9eca

Reply:
Tested on Linux and it works perfect. In theory should do the same on Windows

Reply:
Tested on Windows and works fine:

```java
F:\tdl-runner-java>record_screen_and_upload.bat

Displaying operating system specific systeminfo...
OS Name:                   Microsoft Windows 10 Enterprise Evaluation
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
BIOS Version:              innotek GmbH VirtualBox, 01-Dec-06

JAVA_HOME=C:\PROGRA~1\Java\JDK-11~1.1

JAVA_EXE=C:\PROGRA~1\Java\JDK-11~1.1\bin\java.exe

F:\tdl-runner-java>for /F "tokens=3" %g in ('C:\PROGRA~1\Java\JDK-11~1.1\bin\java.exe -version 2>&1 | findstr /i "version"') do (set JAVA_FULL_VERSION=%g )

F:\tdl-runner-java>(set JAVA_FULL_VERSION="11.0.1" )

JAVA_FULL_VERSION=11.0.1

JAVA_VERSION=11
"--- Java 9 or higher detected (version 11) ---"
"Using DEFAULT_JVM_OPTS variable with value ''"
"--------------------------------------------------------------------------------------------------------------"

F:\tdl-runner-java>"C:\PROGRA~1\Java\JDK-11~1.1\bin\java.exe"    -jar "F:\tdl-runner-java\\record\record-and-upload-capsule.jar" --config F:\tdl-runner-java\\config\credentials.config --store F:\tdl-runner-java\\record\localstore --sourcecode F:\tdl-runner-java\
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by Capsule (file:/F:/tdl-runner-java/record/record-and-upload-capsule.jar) to field com.sun.jmx.mbeanserver.JmxMBeanServer.mbsInterceptor
WARNING: Please consider reporting this to the maintainers of Capsule
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
INFO  [main]       - Starting recording app
INFO  [main]       - Checking permissions
INFO  [main]       - Logging initialized @3969ms to org.eclipse.jetty.util.log.Slf4jLog
```